File: src/me/ch1/fizzbuzz.py
""" fizzbuzz for 1 to 100
fizz on 3
buzz on 5
fiizzbuzz on 15
"""

for i in range(1, 101):
    if i % 15 == 0:
        print("fizzbuzz")
        continue
    if i % 3 == 0:
        print('fizz')
        continue
    if i % 5 == 0:
        print('buzz')
        continue
    print(i)

File: src/me/ch1/fizzbuzz.rs
/*
fizzbuzz for 1 to 100
fizz on 3
buzz on 5
fiizzbuzz on 15
*/
fn main() {
    for i in 1..101 {
        if i % 3 == 0 && i % 5 == 0 {
            println!("fizzbuzz");
        } else if i % 3 == 0 {
            println!("fizz");
        } else if i % 5 == 0 {
            println!("buzz");
        } else {
            println!("{}", i);
        }
    }
}
